src/DIRAC/Core/Tornado/Server/TornadoREST.py

```diff
@@ -4,13 +4,9 @@
 """
 
 import os
-import inspect
```

Comment:
Why? The `inspect` module is heavily used by this file.

Reply:
It gets imported by

    from DIRAC.Core.Tornado.Server.private.BaseRequestHandler import *

It is better to import everything explicitly, I will change it